static-build/components/TestResultSnippet/index.tsx

```diff
@@ -28,8 +28,13 @@ const TestResultSnippet: FunctionalComponent<Props> = ({
       <a href={`#${name}`} title={`Explain the ${result}`} class={$toolName}>
         {name}
       </a>
-      <figure class={$toolIcon}>
-        <img src={toolImages[name as BuildTool]} alt="" />
+      <figure>
+        <a href={`#${name}`} class={$toolIcon}>
+          <img
+            src={toolImages[name as BuildTool]}
+            alt={`${name} bundler logo`}
```

Comment:
```suggestion
            alt={`${name}`}
```

Reply:
Minor tweak. In terms of accessibility the image is representing the build tool. It doesn't seem necessary to announce that it's the icon.